integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java

```diff
@@ -169,7 +190,9 @@ public void testOperatorUpgradeFrom3_0_2(@Namespaces(3) List<String> namespaces)
    */
   @AfterEach
   public void tearDown() {
-    if (System.getenv("SKIP_CLEANUP") == null) {
+    if (System.getenv("SKIP_CLEANUP") == null
+        || (System.getenv("SKIP_CLEANUP") != null
```

Comment:
what is the point of checking for both null and not null? If its null isn't it the next statement going to throw NPE?

Reply:
clarified on slack